src/gatk_gcnv_caller.sh

```
@@ -0,0 +1,655 @@
+#!/bin/bash
+
+# Exit at any point if there is any error and output each line as it is executed (for debugging)
+set -exo pipefail
+
+# prefixes all lines of commands written to stdout with datetime
+PS4='\000[$(date)]\011'
+export TZ=Europe/London
+
+# set frequency of instance usage in logs to 10 seconds
+kill $(ps aux | grep pcp-dstat | head -n1 | awk '{print $2}')
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)